packages/sku/package.json

```diff
@@ -123,6 +132,7 @@
     "didyoumean2": "^7.0.2",
     "ensure-gitignore": "^1.1.2",
     "env-ci": "^11.1.0",
+    "esbuild": "^0.24.2",
```

Comment:
Since we're only using `esbuild` for the `Plugin` type, can this be moved to a dev dependency?